docs/source/modules/convection_simple_betts_miller.rst

```diff
@@ -43,26 +45,26 @@ The final three are used to construct the look up table for calculating the lift
 
 :Tmin: The minimum look up table value with the default of :math:`173` :math:`K`.
 :Tmax: The maximum look up table value with the default of :math:`335` :math:`K`.
-:val_inc: The increment used to linearly interpolate within the look up table with the defaul of :math:`0.01`.
+:val_inc: The increment used to linearly interpolate within the look up table with the default of :math:`0.01`.
 
 
 Diagnostics
 -----------
-The diagnostics are not sent out within the call to convection but are sent out from ``idealized_moist_phys.F90`` with the module name ``atmosphere``. The diagnostics relevant to the simple Betts-Miller scheme are in the table below.
+The diagnostics are not sent out within the call to convection but are sent out from ``idealized_moist_phys.F90`` with the module name ``atmosphere``. The diagnostics relevant to the simple Betts--Miller scheme are in the table below.
 
 +-------------------+----------------------------+------------------------------------+
 | Name              | Description                | Units                              |
 +===================+============================+====================================+
-| dt_qg_convection  | Moisture tendency          |:math:`\frac{kg}{kg}\frac{1}{s}`    |
+| dt_qg_convection  | Moisture tendency          |:math:`kgkg^{-1}s^{-1}`             |
```

Comment:
Thanks for changing these.

Very minor comment, but you can add a tilde between the unit characters to make it look a bit better:
`kg~kg^{-1}~s^{-1}`

Reply:
done